tests/InputFiles/Xcts/HeadOnBns.yaml

```diff
@@ -0,0 +1,192 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: SolveXcts
+# Check: parse
+# Timeout: 30
+
+ResourceInfo:
+  AvoidGlobalProc0: false
+  Singletons: Auto
+
+Background: &background
+  Binary:
+    XCoords: [&x_left -20., &x_right 20.]
+    ObjectA: &star_right
+      TovStar:
+        CentralDensity: 0.0008087415253997405  # h = 1.2
```

Comment:
Where does this and the K come from? In single stars K=100 is standard and gives a maximum mass of the NS that's ~2.2 M_sun. Also, can you use scientific notation here? I find it rather difficult to parse all the leading zeroes. 8.08e-4 is much easier to read. Also, having the central density in terms of g/cm^3 in the comment would be more helpful than the specific enthalpy.

Reply:
Added the reference where these numbers are from, and the other things. Thanks for your review!